src/reducers/pending-breakpoints.js

```diff
@@ -62,6 +65,9 @@ function update(
     }
 
     case "REMOVE_BREAKPOINT": {
+      if (action.breakpoint.hidden) {
```

Comment:
I am not sure I understand why we are doing a no-op here. I think we might want to be able to remove hidden breakpoints. For example, if we have a continue-to-here breakpoint, we will probably want to remove it immediately so that it doesnt trigger the next time it runs, as it is not a user added breakpoint.

Thinking about it a bit further -- perhaps these hidden breakpoints should be "single execution".

Thoughts?

Reply:
Ah, i just realized that this is persisted-breakpoints reducer. i was thinking of this as the breakpoints reducer. ignore this comment!